personality-insights/src/main/java/com/ibm/watson/developer_cloud/personality_insights/v3/PersonalityInsights.java

```diff
@@ -25,135 +22,115 @@
-      request.query(CONSUMPTION_PREFERENCES, options.consumptionPreferences());
+    builder.header("content-type", profileOptions.contentType());
+    if (profileOptions.rawScores() != null) {
+      builder.query("raw_scores", String.valueOf(profileOptions.rawScores()));
     }
-
-    if (options.language() != null) {
-      request.header(HttpHeaders.CONTENT_LANGUAGE, options.language());
+    if (profileOptions.csvHeaders() != null) {
+      builder.query("csv_headers", String.valueOf(profileOptions.csvHeaders()));
     }
-
-    if (options.acceptLanguage() != null) {
-      request.header(HttpHeaders.ACCEPT_LANGUAGE, options.acceptLanguage());
+    if (profileOptions.consumptionPreferences() != null) {
+      builder.query("consumption_preferences", String.valueOf(profileOptions.consumptionPreferences()));
     }
-
-    return request;
-  }
-
-  /**
-   * Accepts text and responds with a {@link Profile} with a tree of characteristics that include personality, needs,
-   * and values. <br>
-   * <br>
-   *
-   * @param text Text to analyze
-   *
-   * @return The personality {@link Profile}
-   */
-  public ServiceCall<Profile> getProfile(final String text) {
-    Validator.notEmpty(text, "text cannot be null or empty");
-
-    final ProfileOptions options = new ProfileOptions.Builder().text(text).build();
-    final RequestBuilder requestBuilder = buildProfileRequest(options);
-    return createServiceCall(requestBuilder.build(), ResponseConverterUtils.getObject(Profile.class));
-  }
-
-  /**
-   * Returns a {@link Profile} with a tree of characteristics that include personality, needs, and values. <br>
-   * <br>
-   * Here is an example of how to get the personality profile given some {@link ProfileOptions}:
-   *
-   * @param options the {@link ProfileOptions}
-   * @return The personality {@link Profile}
-   */
-  public ServiceCall<Profile> getProfile(final ProfileOptions options) {
-    final RequestBuilder requestBuilder = buildProfileRequest(options);
-    return createServiceCall(requestBuilder.build(), ResponseConverterUtils.getObject(Profile.class));
-  }
-
-  /**
-   * Returns a CSV profile.
-   *
-   * @param options the {@link ProfileOptions}
-   * @param includeHeaders if true returns the CSV headers
-   * @return the CSV profile
-   */
-  public ServiceCall<String> getProfileAsCSV(final ProfileOptions options, final boolean includeHeaders) {
-    final RequestBuilder requestBuilder = buildProfileRequest(options);
-
-    requestBuilder.header(HttpHeaders.ACCEPT, HttpMediaType.TEXT_CSV);
-    requestBuilder.query(HEADERS, includeHeaders);
-    return createServiceCall(requestBuilder.build(), ResponseConverterUtils.getString());
+    if (profileOptions.contentType().equalsIgnoreCase(ProfileOptions.ContentType.APPLICATION_JSON)) {
+      builder.bodyJson(GsonSingleton.getGson().toJsonTree(profileOptions.content()).getAsJsonObject());
+    } else {
+      builder.bodyContent(profileOptions.body(), profileOptions.contentType());
```

Comment:
The problem is that we are generating the code based on the swagger file @neil-boyette-ibm. If we were writing it by hand it will be easier to just keep this method.